src/CanvasClass/CanClass.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+class CanClass extends React.Component {
+    componentDidMount() {
+      const canvas = this.refs.canvas
+      const ctx = canvas.getContext("2d")
+      const img = this.refs.image
+      ctx.fillStyle = "#3ab2d6";
+      ctx.fillRect(0,0,200,90)
+      ctx.fillStyle = "#c4d7dd";
+      ctx.font = "20px Courier"
+      ctx.fillText(this.props.playerName, 50, 50)
+      
+    }
+    render() {
+        
+      return(
+        <div>
+          <canvas ref="canvas" width={200} height={100}/>
+          
+        </div>
+      )
+    }
+  }
+  export default CanClass
```

Comment:
The name of the file should be the same as the name of the folder (it is a usual convention). Then it should be `CnvasClass/CanvasClass.js`

Reply:
Yes, I should've done that. I guess it slipped my mind. :)
Sorry for the late reply, but I was kind of really sick.